docs/gettingstarted/structuring.md

```diff
@@ -14,49 +14,65 @@ Pick a unique package name. If you own a URL associated with your project, you c
 
 After the top level package (if you have one) you append a unique name for your mod, such as `examplemod`. In our case it will end up as `com.example.examplemod`.
 
-The `mcmod.info` file
+The `mods.toml` file
 -------------------
 
-This file defines the metadata of your mod. Its information may be viewed by users from the main screen of the game through the Mods button. A single info file can describe several mods. When a mod is annotated by the `@Mod` annotation, it may define the `useMetadata` property, which defaults to `false`. When `useMetadata` is `true`, the metadata within `mcmod.info` overrides whatever has been defined in the annotation.
-
-The `mcmod.info` file is formatted as JSON, where the root element is a list of objects and each object describes one modid. It should be stored as `src/main/resources/mcmod.info`. A basic `mcmod.info`, describing one mod, may look like this:
-
-    [{
-      "modid": "examplemod",
-      "name": "Example Mod",
-      "description": "Lets you craft dirt into diamonds. This is a traditional mod that has existed for eons. It is ancient. The holy Notch created it. Jeb rainbowfied it. Dinnerbone made it upside down. Etc.",
-      "version": "1.0.0.0",
-      "mcversion": "1.10.2",
-      "logoFile": "assets/examplemod/textures/logo.png",
-      "url": "minecraftforge.net/",
-      "updateJSON": "minecraftforge.net/versions.json",
-      "authorList": ["Author"],
-      "credits": "I'd like to thank my mother and father."
-    }]
-
-The default Gradle configuration replaces `${version}` with the project version, and `${mcversion}` with the Minecraft version, but *only* within `mcmod.info`, so you should use those instead of directly writing them out. Here is a table of attributes that may be given to a mod, where `required` means there is no default and the absence of the property causes an error. In addition to the required properties, you should also define `description`, `version`, `mcversion`, `url`, and `authorList`.
+This file defines the metadata of your mod. Its information may be viewed by users from the main screen of the game through the Mods button. A single info file can describe several mods.
+
+The `mods.toml` file is formatted as TOML, the example mods.toml file in the MDK provides comments explaining the contents of the file. It should be stored as `src/main/resources/META-INF/mods.toml`. A basic `mods.toml`, describing one mod, may look like this:
+
+    # The name of the mod loader type to load - for regular FML @Mod mods it should be javafml
```

Comment:
You can surround this in code tags to give it syntax highlighting like so
\`\`\`toml
\<contents>
\`\`\`